src/screens/Notification.js

```diff
@@ -0,0 +1,314 @@
+        </Property>
+
+        <Property name="notification.iconContainer">
+          <Description>Any valid Box prop for the icon container.</Description>
+          <PropertyValue type="object">
+            <Example defaultValue>
+              {`
+                    {
+                        pad: { right: 'small' },
+                    }
+                `}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.textContainer">
+          <Description>Any valid Box prop for the text container.</Description>
+          <PropertyValue type="object">
+            <Example defaultValue>
+              {`
+                    {
+                        gap: 'medium',
+                    },
+                `}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.title">
+          <Description>Any valid Text prop for the title text.</Description>
+          <PropertyValue type="object">
+            <Description>
+              'color' is used for the text color and 'weight' for the weight of
+              the title.
+            </Description>
+            <Example defaultValue>
+              {`
+                    {
+                        weight: 'bold',
+                        color: {
+                            light: 'black',
+                            dark: 'white',
+                        },
+                    },
+                `}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.message">
+          <Description>Any valid Text prop for the message text.</Description>
+          <PropertyValue type="object">
+            <Description>'color' is used for the message color.</Description>
+            <Example defaultValue>
+              {`
+                    {
+                        color: {
+                            light: 'black',
+                            dark: 'white',
+                        },
+                    },
+                `}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.button.icon">
+          <Description>The default close button icon.</Description>
+          <PropertyValue type="node">
+            <Example defaultValue>{`<FormClose />`}</Example>
+            <Example>{`<Icon />`}</Example>
+          </PropertyValue>
+          <PropertyValue type="SVG">
+            <Example>{`<svg>...</svg>`}</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.button.color">
+          <Description>The color of the close button icon.</Description>
+          <GenericColor />
+          <PropertyValue>
+            <Example defaultValue>
+              {`
+                    color: {
+                        light: 'black',
+                        dark: 'white',
+                    },`}
+            </Example>
```

Comment:
Same alignment comment as above, also remove trailing comma and remove `color:`. In this example you don't need both `{ dark: "string", light: "string" }` and `{ light: 'black', dark: 'white' }` just show the default value for the object example.